src/main/java/seedu/address/model/lesson/ModuleCode.java

```diff
@@ -0,0 +1,58 @@
+package seedu.address.model.lesson;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents an assignment's module code in the address book.
```

Comment:
Perhaps comment can be changed to just 'module code'? Since it is applicable to both lessons and assignments

Reply:
Amended. Changed to Task instead. :)